src/sortStudents.ts

```diff
@@ -1,16 +1,68 @@
 
 export interface Student {
-  // describe Student interface
+  name: string;
+  surname: string;
+  age: number;
+  married: boolean;
+  grades: number[];
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name,
+  Surname,
+  Age,
+  Married,
+  AverageGrade
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+function calculateAverage(gradesArray: number[]): number {
```

Comment:
It is better to give clear naming. Average what?
```suggestion
function calculateAverageGrade(gradesArray: number[]): number {
```